tests/kafkatest/services/kafka/kafka.py

```diff
@@ -277,6 +278,9 @@ def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAI
         self.isolated_controller_quorum = None # will define below if necessary
```

Comment:
make sure you add a description of the param you are adding to the docstring for the __init__ method